compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1396,11 +1396,9 @@ class Namer { typer: Typer =>
         WildcardType
     }
     val memTpe = paramFn(checkSimpleKinded(typedAheadType(mdef.tpt, tptProto)).tpe)
-    if (ctx.explicitNulls && mdef.mods.is(JavaDefined)) {
+    if (ctx.explicitNulls && mdef.mods.is(JavaDefined))
```

Comment:
I'm not a big fan of removing the braces in cases like this, because somebody  might later add a statement in the body of the then branch, and things will break unexpectedly (granted, with a compile time error). 

But ok...

Reply:
Is this documented in their style guide?

Reply:
I can't find any document about the code style. I can only find the comment similar to this: https://github.com/lampepfl/dotty/pull/6344#discussion_r299975937

Reply:
Ok looks good then.